src/apps/kwenta/optimism/kwenta.cross.contract-position-fetcher.ts

```diff
@@ -0,0 +1,38 @@
+import { OptimismKwentaPerpContractPositionFetcher } from './kwenta.perp.contract-position-fetcher';
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+import { gql } from 'graphql-request';
+import KWENTA_DEFINITION from '../kwenta.definition';
+
+type GetCrossMarginAccounts = {
+  crossMarginAccounts: {
+    id: string;
+  }[];
+};
+
+const getCrossMarginAccountsQuery = gql`
+  query MyQuery($address: String!) {
+      crossMarginAccounts(where: {owner: $address}) {
+        id
+      }
+  }
+`;
+
+@PositionTemplate()
+export class OptimismKwentaCrossContractPositionFetcher extends OptimismKwentaPerpContractPositionFetcher {
+
+  groupLabel = 'Cross Margin';
+  groupId = KWENTA_DEFINITION.groups.cross.id;
```

Comment:
groupId isn't necessary with @PositionTemplate() since it sets appId, groupId and network based on the name of the file.

want to see by yourself ? Have a look at `src/app-toolkit/decorators/position-template.decorator.ts`